pkgs/applications/misc/seatd/default.nix

```diff
@@ -18,13 +19,13 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-JwlJLHkRgSRqfQEhXbzuFTmhxfbwKVdLICPbTDbC9M0=";
   };
 
-  nativeBuildInputs = [ meson ninja pkg-config ];
+  outputs = [ "bin" "out" ];
```

Comment:
You are sure we do not want dev here, too?

Reply:
Yeah makes sense.  I've added it, and a man output.